src/main/drivers/bus.h

```diff
@@ -19,14 +19,21 @@
 
 #include "platform.h"
 
+#include "drivers/bus_i2c.h"
 #include "drivers/io_types.h"
 
 typedef union busDevice_u {
```

Comment:
`busDevice_u` should carry its type, it will help with dual-interface devices where driver bus can be easily abstracted and will improve error checking.
Anonymous union inside struct will be easier to read (available from -std=c11)